go/minterm/minterm.go

```diff
@@ -37,6 +40,7 @@ func New() (*MinTerm, error) {
 
 // Shutdown closes the terminal.
 func (m *MinTerm) Shutdown() error {
+	m.restore()
```

Comment:
If we spawned a Prompt call in a goroutine, which I don't think we ever do. 

But I can add code to protect it to be safe.